CHANGELOG.md

```diff
@@ -4,8 +4,12 @@ Releases
 v1.15.0-dev (unreleased)
 --------------------
 
--   No changes yet.
-
+- yarpcerrors: Change the contract for ErrorCode and ErrorMessage
```

Comment:
Word this in a way that doesn't sound like a breaking change.

Reply:
Also, should say what happened, not just "change something"

Reply:
How about now?